src/screens/FeedScreen/RenderList.js

```diff
@@ -51,7 +51,8 @@ const RenderListFeed = (props) => {
     onDeletePost,
     isShowDelete,
     isSelf,
-    onHeaderOptionClicked = () => {}
+    onHeaderOptionClicked = () => {},
+    isFromFeeds = false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `isFromFeeds` prop with a default value of `false` is a good practice. It ensures that if this prop is not provided, the component will still function correctly without throwing an error due to undefined variables. This change improves the robustness of the code.
```diff
+    isFromFeeds = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->